src/components/voyage/VoyageLog.tsx

```diff
@@ -126,17 +127,16 @@ export const VoyageLog = (props:{}) => {
          <br />
       </div>;
 
-   function getColumns() {
-      const spriteClass = GetSpriteCssClass();
+   function getColumns(spriteClass: string) {
       return [
          {
             id: 'icon',
             Header: '',
-            minWidth: 30,
-            maxWidth: 30,
+            minWidth: 72,
+            maxWidth: 72,
             resizable: false,
             accessor: (row: any) => row.full_name,
-            Cell: (p: any) => <img className={spriteClass} src={p.original.iconUrl} height={25} />
+            Cell: (p: any) => <img className={`image-fit ${spriteClass}`} src={p.original.iconUrl} />
```

Comment:
The icons in the reward table are too large. 25-30px would be better than the original image size, which sometimes varies.